tests/recovery.py

```diff
@@ -993,6 +995,57 @@ def run_recover_snapshot_alone(args):
         return network
 
 
+def run_recover_via_initial_recovery_owner(args):
```

Comment:
This seems unnecessary? I think we could remove the `initial_recovery_owner_count` entirely, and associated CLI arg. We can test the flow for adding recovery owners _after_ network start (or upgrading an existing member to recovery owner), which is what most actual services doing this must do? They won't be launched with our test infra.

Reply:
We needed the flexibility to specify a recovery owner as part of service creation itself. So it would be good to keep the test case for it.

Reply:
@eddyashton if we support it in the config, which I think is the consistent thing to do because we support member_data, then we must test it.

As discussed, there could be a smaller configuration option, where perhaps only a single member key is allowed (the operator), and all preparation of the network (definition of the consortium, constitution etc), must be done through governance steps. It could even be a single operator key, _and_ a small, hardcoded constitution that CCF defines, enough for anything to bootstrap to something larger. This would be pushing complexity out of the implementation and into operations, which is attractive from an implementation and security perspective, but probably not from an operations perspective.

I think we're probably going the other way, towards auto-opening networks, for foundational use cases. But I can see the appeal for multi-party consortiums.